src/styles/_general.scss

```diff
@@ -22,6 +22,12 @@
 	min-height: 100vh;
 }
 
+.min-vh-lg-100 {
+	@include media-breakpoint-up(lg) {
+		min-height: 100vh;
```

Comment:
Am I reading correctly that this makes the header the full height of the viewport on large screens? Seems pretty extreme.